.github/workflows/release.yml

```diff
@@ -158,22 +164,29 @@ jobs:
           echo "SERVICE_NAME: $SERVICE_NAME"
           echo "::set-output name=service_name::$SERVICE_NAME"
 
-      - name: Pull latest selfhost Release image
-        run: docker pull bitwarden/${{ steps.setup.outputs.service_name }}:latest
+      - name: Pull latest selfhost RC image
```

Comment:
Maybe we can remove the "RC" here since it might not be the `rc` branch, even though the `hotfix` branch is a release candidate?